### PR TITLE
add role="listitem"

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata PTAL

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "core-doc-viewer": "Polymer/core-doc-viewer#0.8-preview",
     "core-icon": "Polymer/core-icon#0.8-preview",
     "core-icons": "Polymer/core-icons#0.8-preview",
+    "test-fixture": "Polymer/test-fixture#master",
```
Comment (cdata) [inline bower.json]: There's a 0.0.2 tag that you can use for this. Might be better?

**Comment by cdata** [comment]
Dependency comment, otherwise LGTM :+1:

File: bower.json (hunk 1 above)
Comment (morethanreal) [inline bower.json]: My lack of bower-fu is showing but I don't really understand how we should deal with components depending on different versions? For example, if this element has `Polymer/test-fixture#^0.0.2` and somebody else has `Polymer/test-fixture#master`, is that going to cause problems?

Comment (cdata) [inline bower.json]: Yeah, that will create an issue where bower will prompt you to pick one of the two. What do you think we should do about that?

Comment (morethanreal) [inline bower.json]: In principle I think we should specify a version, but in practice is the plan to update all dependent elements when we rev the version of some element?

Comment (cdata) [inline bower.json]: I think that's a good practice. It is made painful by the way that we treat elements (as separate projects in different repos). However, the elements are an explicitly interdependent body of code. If Polymer has a new version, and we want to update an element to that version, then all of that element's dependencies should also be updates where necessary.

And an update: I have tagged 0.8.0 in polymerelements/test-fixture. You should be able to depend on that reliably.

Comment (morethanreal) [inline bower.json]: Cool, I'm updating to use it.

**Pull request merged**